### PR TITLE
Sync Template and Cluster YAMLs

### PR DESCRIPTION
**Pull request by alfredoihernandez**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Update template yamls from cluster yamls to ensure consistency. Keep checkov changes where applicable.
Also includes renaming template files to ensure naming consistency. Required changes to READMEs to accomodate file name changes

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/850

### CODE REVIEW

File: templates/fluentbit/05-daemonset.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
         - name: fluentbit
-          image: $ASB_ACR_NAME.azurecr.io/fluent/fluent-bit:1.9.5
+          image: acraks3i2qzkkxofr7c.azurecr.io/fluent/fluent-bit:1.9.5
```
Comment (joaquinrz) [inline templates/fluentbit/05-daemonset.yaml]: should be $ASB_ACR_NAME

Comment (alfredoihernandez) [reply templates/fluentbit/05-daemonset.yaml]: done

File: templates/monitoring/grafana/05-grafana.yaml [hunk 2]
```diff
@@ -152,4 +163,3 @@ spec:
         port:
           number: 3000
         host: grafana
```
Comment (alfredoihernandez) [inline templates/monitoring/grafana/05-grafana.yaml:165]: not sure why this is not showing in my editor
```suggestion
        host: grafana

```

File: monitoring/README.md [hunk 3]
```diff
@@ -71,11 +71,23 @@ export ASB_GRAFANA_SP_CLIENT_ID=[insert CLIENT_ID]
...
+# create grafana-datasources config-map file
+cat templates/monitoring/grafana/02-grafana-config-datasources.yaml | envsubst  > $ASB_GIT_PATH/monitoring/grafana/02-grafana-config-datasources.yaml
+
+# create grafana-dashboard-providers config-map file
+cat templates/monitoring/grafana/03-grafana-config-dashboards.yaml | envsubst  > $ASB_GIT_PATH/monitoring/grafana/03-grafana-config-dashboards.yaml
+
+# create AzureIdentity, AzureIdentityBinding, SecretProviderClass file
+cat templates/monitoring/grafana/04-grafana-pod-identity.yaml | envsubst  > $ASB_GIT_PATH/monitoring/grafana/04-grafana-pod-identity.yaml
+
+# IMPORTANT: dashboard yaml files are not in templates
```
Comment (alfredoihernandez) [inline monitoring/README.md:86]: Added triage item to address this: https://github.com/orgs/retaildevcrews/projects/12#card-84152204

**Review by sivamu — APPROVED** [review]

**Pull request merged**